java/src/main/native/src/AggregationJni.cpp

```diff
@@ -186,10 +186,10 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_Aggregation_createLeadLagAgg(JNIEnv
     std::unique_ptr<cudf::aggregation> ret;
     // These numbers come from Aggregation.java and must stay in sync
     switch (kind) {
-      case 19: // LEAD
+      case 20: // LEAD
```

Comment:
Is there any way to compare the `kind` value directly with the value from enum `Kind`? For example:
```
case Kind.LEAD:
...
```

Reply:
Hi @ttnghia, I replaced the pattern match on raw number with pattern match on ``aggregation::Kind`. Is it what you would like to do?

Reply:
Sorry I don't see any change here. But that's fine as it is outside of scope of this PR.